product_attribute_model_link/views/product_attribute_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_attribute_model_link_form" model="ir.ui.view">
+        <field name="name">product.attribute.model.link.form</field>
+        <field name="model">product.attribute</field>
+        <field name="inherit_id" ref="product.product_attribute_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='main_fields']" position="after">
+                <group>
+                    <group name="model_link_fields">
+                        <field name="linked_model_id" />
+                        <field
+                            name="linked_field_id"
+                            attrs="{'required': [('linked_model_id', '!=', False)]}"
+                        />
+                        <field name="domain" />
+                        <field name="apply_to_products_on_create" />
+                        <field name="create_from_attribute_values" />
+                        <field
+                            name="modify_from_attribute_values"
+                            attrs="{'invisible': [('create_from_attribute_values', '=', False)]}"
+                        />
+                        <field
+                            name="delete_when_attribute_value_is_deleted"
+                            attrs="{'invisible': [('create_from_attribute_values', '=', False)]}"
+                        />
+                    </group>
+                </group>
+            </xpath>
+            <xpath expr="//group[@name='model_link_fields']" position="before">
+                <xpath expr="//group[@name='main_fields']" position="move" />
+            </xpath>
+            <xpath
+                expr="//field[@name='value_ids']/tree/field[@name='name']"
+                position="after"
+            >
+                <field name="linked_record_ref" readonly="1" />
```

Comment:
It's not possible to link existing record to a new attribute value. Because `name` field is mandatory. However if one enters a value in the `name' field it will create a new record in the model. And we don't want to create a new record. We want to use an existing one.
![image](https://github.com/OCA/product-attribute/assets/2234999/92708d0c-23b6-4556-a5f1-27a71d32820a)